documentation/book/proc-configuring-mirror-maker.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+[id='configuring-kafka-bridge-{context}']
+= Configuring Kafka Mirror Maker
+
+Use the properties of the `KafkaMirrorMaker` resource to configure your Kafka Mirror Maker deployment.
+
+You can configure access control for producer and consumer using TLS or SASL authentication.
+This procedure shows a configuration that uses TLS encryption and authentication on the consumer side for access control.
+
+For more information on configuring access control for Kafka Mirror Maker, see xref:assembly-access-configuration-kafka-mirror-maker-{context}[Configuring Kafka Mirror Maker access control].
+
+.Prerequisites
+
+* xref:cluster-operator-str[{ProductName} and Kafka is deployed]
+* Source and target Kafka clusters are available
+
+.Procedure
+
+. Edit the `spec` properties for the `KafkaMirrorMaker` resource.
++
+The properties you can configure are shown in this example configuration:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaMirrorMaker
+metadata:
+  name: my-mirror-maker
+spec:
+  # ...
+  replicas: 3 <1>
+  consumer:
+    bootstrapServers: my-source-cluster-kafka-bootstrap:9092 <2>
+    groupId: "my-group" <3>
+    numStreams: 2 <4>
+    offsetCommitInterval: 120000 <5>
+    tls: <6>
```

Comment:
Could we add the TLS and authentication sections also to the producr to make it clear that these are configured independently?

Reply:
Added